data/brutus-api/MathAndWeatherAndOsu.csv

```diff
@@ -0,0 +1,101 @@
+what is 656 plus 748 ?,math

```

Comment:
yeah it has been working so far. It looks for statements that are similar, not exactly the same